package.json

```diff
@@ -205,6 +205,7 @@
     "node-fetch": "1.7.3",
     "opn": "^5.5.0",
     "oppsy": "^2.0.0",
+    "path-to-regexp": "^1.7.0",
```

Comment:
We depend on this in the code project, but it's not listed in the x-pack package.json, maybe we can list it there?